src/pages/Wallet/AddressDetailsPage.tsx

```diff
@@ -162,4 +235,15 @@ const MainAddress = styled.div`
   position: absolute;
 `
 
+const DarkLabel = styled.span`
```

Comment:
This could actually use the `Badge` component.

Reply:
Updated. I am sure we'll update it again once we re-iterate over the styles of labels and badges.